### PR TITLE
[Fix] - Trim Names in Settings > Members table #7509

### PR DESCRIPTION
**Pull request by Karankhatik**
Issue: Long names in the Members table were overflowing, affecting the layout.

Fix:
- Trimmed long names with ellipses.
- Added tooltips to display the full content on hover.
- Max-width of the text dynamically set to 90px on large screens, and 60px on mobile.
image

### CODE REVIEW

**Comment by Bonapara** [comment]
Current:

CleanShot 2024-10-09 at 14 32 20

It looks like there is an undesired right padding. The gap between the name and the email should be only 16px

Desired:

CleanShot 2024-10-09 at 14 36 26

**Comment by Karankhatik** [comment]
Hello @charlesBochet what should I do, should I work on @Bonapara feedback or code is fine. Please confirm.

**Comment by charlesBochet** [comment]
> Hello @charlesBochet what should I do, should I work on @Bonapara feedback or code is fine. Please confirm.

Work on @Bonapara feedbacks :)

**Comment by Karankhatik** [comment]
@Bonapara I've successfully removed the undesired space; however, there's an issue where, if the name is shorter then the  (ellipsis length), the email gets too close to the name, which affects the layout. Please provide your suggestions.

image
image

**Comment by Bonapara** [comment]
@Karankhatik The issue isn't the spacing between the fields, which should remain the same. It's the name field that isn't using its full available width.

**Comment by charlesBochet** [comment]
You also have conflicts now :)

**Comment by Karankhatik** [comment]
@Bonapara, you mean the text should take up the entire width of the container, and only when it exceeds the container's width, we apply an ellipsis? Also, the space between the name and email will be 16px. Is that correct?

**Comment by Bonapara** [comment]
Absolutely!

**Comment by Karankhatik** [comment]
Thanks will update u ASAP.

File: packages/twenty-front/src/pages/settings/SettingsWorkspaceMembers.tsx [hunk 1]
```diff
@@ -97,6 +103,26 @@ const StyledTableHeaderRow = styled(Table)`
   margin-bottom: ${({ theme }) => theme.spacing(1.5)};
 `;
 
+const StyledTableRowForMemberList = styled(TableRow)`
+  gap: 16px;
+  @media (max-width: 768px) {
```
Comment (charlesBochet) [inline packages/twenty-front/src/pages/settings/SettingsWorkspaceMembers.tsx:108]: all these cases look too heavy, I'm taking a look

**Comment by charlesBochet** [comment]
Mobile:
image

**Comment by charlesBochet** [comment]
Desktop: 
image

**Review by charlesBochet — APPROVED** [review]
I've reworked a bit to simplify CSS

**Pull request merged**